packages/react/src/components/TableOfContents/README.md

```diff
@@ -0,0 +1,97 @@
+# Sticky Table of Contents
+
+> The Sticky Table of Contents component is to be utilized within IBM.com.
+
+## Getting started
+
+Here's a quick example to get you started.
+
+```javascript
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { TableOfContents } from '@carbon/ibmdotcom-react';
+import '@carbon/ibmdotcom-styles/scss/components/tableofcontents/index.scss';
+
+function App() {
+  return (
+    <TableOfContents menuLabel={menuLabel} menuItems={menuItems}>
+      <a name="8"></a>
+      <h3 style={{ paddingBottom: '1rem', paddingTop: '2rem' }}>
+        Cras molestie condimentum
+      </h3>
+      <p>
+        Elementum dui gravida non. Mauris et nisl semper, elementum quam non,
+        lacinia purus. Vivamus aliquam vitae sapien volutpat efficitur.
+      </p>
+
+      <a name="7"></a>
+      <h3 style={{ paddingBottom: '1rem', paddingTop: '2rem' }}>
+        Praesent fermentum sodales
+      </h3>
+      <p>
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras molestie
+        condimentum consectetur. Nulla tristique lacinia elit.
+      </p>
+    </TableOfContents>
+  );
+}
+
+ReactDOM.render(<App />, document.querySelector('#app'));
+```
+
+> 💡 Don't forget to import the tableofcontents styles from
+> [@carbon/ibmdotcom-styles](https://github.com/carbon-design-system/ibm-dotcom-library/blob/master/packages/styles).
+
+#### Feature Flags
+
+To utilize the following features, set the following variable's to `true` within
+your `.env` file or your application build settings.
+
+```
+TABLE_OF_CONTENTS=true
```

Comment:
rename to `DDS_TOC`, or `DDS_TABLE_OF_CONTENTS`

Reply:
I like a shorter one.